docs/adoc/intro-overview.adoc

```diff
@@ -0,0 +1,37 @@
+include::1-attribute.adoc[]
+
+[#intro-overview]
+# Overview of {ds2}
+
+{ds2} is a tool to publish large-scale documentation websites accommodating multiple products, product versions, and localizations.
+It focuses on allowing publication of {db}/{ad} content that is compatible with {daps}.
+However, it can also accommodate links to other sources of documents.
+
+The {ds2} build server is written in Python/Bash/XSLT and designed to run as a daemon.
+Its Git repository is at https://github.com/openSUSE/docserv.
+
+image::docserv2-overview.svg[title="{ds2} overview",width=90%]
+
+The following elements that are relevant to {ds2}:
+
+* Machines:
+
+** Git is used for versioning the sources of the content to publish.
+
+** The configuration for {ds2} should be stored in a Git repository as well, though a separate one to that of the documentation.
```

Comment:
Git and configuration files grouped under "Machines"? Hmn, I don't get that. Actually, it confused me. Maybe find a different category for these two points?

What about "Version control config files"? Or, if you prefer it more general, "Basics"?

Reply:
Removed the point.